package.json

```diff
@@ -24,11 +24,12 @@
     "expose-loader": "0.7.x",
     "express": "4.13.3",
     "imports-loader": "0.6.5",
-    "nodemon": "1.8.1",
     "karma": "^0.12.37",
+    "karma-chrome-launcher": "^0.2.2",
```

Comment:
Please add an exact version (e.g. no preceding ^) so that we keep things more deterministic.  Feel free to also remove the other ^ lines in package.json while you're at it.  :-)